src/rpc/rpc_schema.js

```diff
@@ -68,12 +73,12 @@ class RpcSchema {
             });
         });
         try {
-            this._ajv.addSchema(api);
+            this._ajv.addSchema(api, api.$id);
```

Comment:
do you really need to pass the second arg $id here? I thought that addSchema will take $id from the schema anyhow